src/plugins/intel_gna/CMakeLists.txt

```diff
@@ -41,10 +41,12 @@ file(GLOB_RECURSE HEADERS
         ${CMAKE_CURRENT_SOURCE_DIR}/src/*.h
         ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp)
 
-# build avx2.cpp with AVX2 support
-ie_avx2_optimization_flags(avx2_flags)
-set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/src/avx2.cpp PROPERTIES COMPILE_OPTIONS "${avx2_flags}")
-add_compile_definitions(HAVE_AVX2=1)
+# build avx2.cpp with AVX2 support, only for Windows
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
```

Comment:
why don't you use common OpenVINO's `ENABLE_AVX2` ?

Reply:
Right, I forgot to add this condition. Fixing this now.
The "compiler == MSVC" condition has to stay here, however, because we decided that Linux won't be supported for now - I had too many problems trying to make it work and it's not a strict requirement